README.md

```diff
@@ -8,6 +8,26 @@ Download the repository to your computer:
 git clone https://github.com/cta-observatory/2022_01_lstchain_school.git
 ```
 
+## Install
+
+- You will need to install [miniconda](https://docs.conda.io/en/latest/miniconda.html) (recommended) or [anaconda](https://www.anaconda.com/distribution/#download-section) first. 
```

Comment:
I would actually recommend `mamba-forge`, so mamba is already in the base env:

https://github.com/conda-forge/miniforge#mambaforge

Reply:
done